src/@chakra-ui/gatsby-plugin/components/Tag.ts

```diff
@@ -0,0 +1,44 @@
+import type { ComponentStyleConfig } from "@chakra-ui/theme"
+
+export const Tag: ComponentStyleConfig = {
+  parts: ["container", "label"],
+  baseStyle: {
+    container: {
+      display: "flex",
+      alignItems: "center",
+      bgImage:
```

Comment:
For some reason, this `bg` is not being applied on dark mode.
![image](https://user-images.githubusercontent.com/468158/188968537-29aabc97-064c-4c88-b8e9-f83f537da5f4.png)

Reply:
It was caused by the default style of `variant="subtle"` props. I figured a better practice that is passing a `custom` value  into `variant` to avoid the style pollution.